Specs/Core/loadKTX2Spec.js

```diff
@@ -664,198 +276,90 @@ describe("Core/loadKTX2", function () {
-    expect(rejectedError.statusCode).toBeUndefined();
-    expect(rejectedError.response).toBeUndefined();
   });
 
-  it("returns a promise that rejects when the request results in an HTTP error code", function () {
-    var testUrl = "http://example.invalid/testuri";
-    var promise = loadKTX2(testUrl);
-
-    expect(promise).toBeDefined();
-
-    var resolvedValue;
-    var rejectedError;
-    promise.then(
-      function (value) {
-        resolvedValue = value;
-      },
-      function (error) {
-        rejectedError = error;
-      }
-    );
-
-    expect(resolvedValue).toBeUndefined();
-    expect(rejectedError).toBeUndefined();
-
-    var error = "some error";
-    fakeXHR.simulateHttpResponse(404, error);
-    expect(resolvedValue).toBeUndefined();
-    expect(rejectedError).toBeInstanceOf(RequestErrorEvent);
-    expect(rejectedError.statusCode).toEqual(404);
-    expect(rejectedError.response).toEqual(error);
-  });
-
-  it("returns a promise that resolves with undefined when the status code is 204", function () {
-    var testUrl = "http://example.invalid/testuri";
-    var promise = loadKTX2(testUrl);
-
-    expect(promise).toBeDefined();
-
-    var resolved = false;
-    var resolvedValue;
-    var rejectedError;
-    promise.then(
-      function (value) {
-        resolved = true;
-        resolvedValue = value;
-      },
-      function (error) {
-        rejectedError = error;
-      }
-    );
-
-    expect(resolvedValue).toBeUndefined();
-    expect(rejectedError).toBeUndefined();
-
-    fakeXHR.simulateHttpResponse(204);
-    expect(resolved).toBe(true);
-    expect(resolvedValue).toBeUndefined();
-    expect(rejectedError).toBeUndefined();
-  });
-
-  it("returns a promise that resolves to an uncompressed texture when the request loads", function () {
-    var testUrl = "http://example.invalid/testuri";
-    var promise = loadKTX2(testUrl);
-
-    expect(promise).toBeDefined();
-
-    var resolvedValue;
-    var rejectedError;
-    promise.then(
-      function (value) {
-        resolvedValue = value;
-      },
-      function (error) {
-        rejectedError = error;
-      }
-    );
-
-    expect(resolvedValue).toBeUndefined();
-    expect(rejectedError).toBeUndefined();
-
-    var response = validUncompressed.buffer;
-    fakeXHR.simulateLoad(response);
-    expect(resolvedValue).toBeDefined();
-    expect(resolvedValue.width).toEqual(4);
-    expect(resolvedValue.height).toEqual(4);
-    expect(
-      PixelFormat.isCompressedFormat(resolvedValue.internalFormat)
-    ).toEqual(false);
-    expect(resolvedValue.bufferView).toBeDefined();
-    expect(rejectedError).toBeUndefined();
+  it("returns a promise that resolves to an uncompressed texture", function () {
+    var resource = Resource.createIfNeeded("./Data/Images/Green4x4.ktx2");
+    var loadPromise = resource.fetchArrayBuffer();
+    return loadPromise.then(function (buffer) {
+      var promise = KTX2Transcoder.transcode(buffer, { pvrtc: true });
```

Comment:
The `{ pvrtc: true }` object is kinda phony, but at least one format needs to be "supported" for the logic in `parseKTX2` to go forward without error-ing. Is there a better way to get the formats here? Should all formats be tested in specs somehow?